yarn.lock

```diff
@@ -0,0 +1,1945 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Get rid of this yarn.lock file please they don't use yarn. Just run npm install for directline.